prow/plugins/cat/cat.go

```diff
@@ -143,26 +141,27 @@ func (r *realClowder) Url(category string) string {
 
 func (r *realClowder) readCat(category string) (string, error) {
 	uri := r.Url(category)
-	req, err := http.NewRequest("GET", uri, nil)
-	if err != nil {
-		return "", fmt.Errorf("could not create request %s: %v", uri, err)
-	}
-	req.Header.Add("Accept", "application/json")
-	resp, err := client.Do(req)
+	resp, err := http.Get(uri)
 	if err != nil {
 		return "", fmt.Errorf("could not read cat from %s: %v", uri, err)
 	}
+	if sc := resp.StatusCode; sc > 299 || sc < 200 {
+		return "", fmt.Errorf("failing %d response from %s", sc, uri)
+	}
 	defer resp.Body.Close()
 	var a catResult
 	if err = xml.NewDecoder(resp.Body).Decode(&a); err != nil {
 		return "", err
 	}
+	if a.Image == "" {
+		return "", fmt.Errorf("no image url in response from %s", uri)
+	}
 	// checking size, GitHub doesn't support big images
 	toobig, err := github.ImageTooBig(a.Image)
 	if err != nil {
-		return "", err
+		return "", fmt.Errorf("could not validate image size %s: %v", a.Image, err)
 	} else if toobig {
-		return "", errors.New("unsupported cat :( size too big: " + a.Image)
```

Comment:
:(

Reply:
please update this error message to "long cat is too long"

Reply:
/hold
this is an important change

Reply:
Done